cardscan-base/src/main/java/com/getbouncer/cardscan/base/ScanActivityImpl.java

```diff
@@ -71,8 +72,15 @@ public void onClick(View v) {
         if (!mInDebugMode) {
             mDebugImageView.setVisibility(View.INVISIBLE);
         }
+
+        boolean showEnterCardManuallyButton = getIntent().getBooleanExtra(ENTER_CARD_MANUALLY_BUTTON, false);
+        TextView enterCardManuallyButton = findViewById(R.id.enterCardManuallyButton);
+        if (!showEnterCardManuallyButton) {
+            enterCardManuallyButton.setVisibility(View.INVISIBLE);
```

Comment:
instead, this should use `View.GONE`, since that completely drops the view from the rendering pipeline. with `View.INVISIBLE`, the view is fully calculated, but not rendered. `INVISIBLE` is good for things like constraint layouts where other components depend on this one.